utils/systemd/daos-server.service

```diff
@@ -1,10 +1,16 @@
 [Unit]
 Description=DAOS Server
-#OnFailure=<mediation services>
+Wants=network-online.target
+After=network-online.target
 
 [Service]
-Restart=always
-#Environment=DAOS_PATH=/usr/lib/daos
-ExecStart=/usr/bin/orterun -np 1 -c 1 --hostfile hostfile --enable-recovery --report-uri /tmp/urifile daos_server
+Type=simple
+RuntimeDirectory=daos_server
+RuntimeDirectoryMode=0755
+#Only specify if you are running out of your development Tree
+#Also comment out the default ExecStart line
+#ExecStart=/<path to>/install/bin/daos_server -t 1 -i
+ExecStart=/usr/bin/daos_server -t 1 -i
```

Comment:
Seems like it'd be better to use the config file rather than hard-coding parameters here and in the agent...

Reply:
It would be but for now until the certificate work is properly documented and tested I force insecure mode. Once we get more testing on it then I'll remove those. We also bind it to just one core for now as well. With starting the server from the commandline you can easily ctrl+c it. If you saturate the whole machine you might not get to issue the systemctl stop daos-server.

Reply:
I should probably remove the -t 1.